consensus/safety-rules/src/safety_rules.rs

```diff
@@ -104,7 +104,8 @@ impl SafetyRules {
         let block1 = proposed_block.quorum_cert().certified_block().round();
         let block0 = proposed_block.quorum_cert().parent_block().round();
 
-        let commit = block0 + 1 == block1 && block1 + 1 == block2;
+        let commit = u64::checked_add(block0, 1).unwrap() == block1
+            && u64::checked_add(block1, 1).unwrap() == block2;
```

Comment:
considering unwrap will cause a panic, I think the correct behavior would be to return a result here and fail safely.

Reply:
I assume that such a failure will get logged somewhere, of course.

Reply:
done!